DBM-Magtheridon/Magtheridon.lua

```diff
@@ -83,7 +83,7 @@ function mod:CHAT_MSG_MONSTER_YELL(msg)
 		-- +18 to the timers
 		timerConflagration:AddTime(18)
 		timerQuake:AddTime(18)
-		timerBlastNovaCD:AddTime(18)
+		timerBlastNovaCD:AddTime(18, self.vb.blastNovaCounter)
```

Comment:
Shouldn't it be this?
```suggestion
		timerBlastNovaCD:AddTime(18, self.vb.blastNovaCounter+1)
```

Reply:
It already gets incremented here.
https://github.com/Zidras/DBM-Warmane/blob/8eb1de1a6a58a3a3074b5d3844c9d1cbd42dce9c/DBM-Magtheridon/Magtheridon.lua#L50-L55

It's just adding 18 seconds to the current timer, so whatever the current bar is. The screenshots above should give you a good example of it.

Reply:
Ahh you started the counter at 1. that's not really standard practice in dbm, but since you don't use for the warnings, I'll leave it be